crud/get.lua

```diff
@@ -24,7 +81,9 @@ local function get_on_storage(space_name, key)
 
     -- add_space_schema_hash is false because
     -- reloading space format on router can't avoid get error on storage
-    return schema.wrap_box_space_func_result(false, space, 'get', key)
```

Comment:
What do you think about delegating `fields` processing to `schema.wrap_box_space_func_result`?

You can change this function sugnature.

Now it is

`function schema.wrap_box_space_func_result(add_space_schema_hash, space, func_name, ...)`

If it become 

`function schema.wrap_box_space_func_result(space, func_name, args, opts)`, 
where `opts` is `{add_space_schema_hash = '?boolean', fields = '?table'}`, it can be used this way:

```lua
return schema.wrap_box_space_func_result(space, 'get', {key}, {
    add_space_schema_hash = false, 
   fields = fields,
})
```